.github/workflows/system-tests.yml

```diff
@@ -300,7 +300,11 @@ jobs:
           docker image list
       - name: Run scenario
         run: |
-          ./run.sh ++docker ${{ matrix.scenario }} ${{matrix.scenario == env.FORCE_TESTS_SCENARIO && env.FORCE_TESTS || ''}}
+          if [ ${{ matrix.scenario }} != "CROSSED_TRACING_LIBRARIES" ]; then
+            ./run.sh ++docker ${{ matrix.scenario }} ${{matrix.scenario == env.FORCE_TESTS_SCENARIO && env.FORCE_TESTS || ''}}
+          else
+            ./run.sh ${{ matrix.scenario }} ${{matrix.scenario == env.FORCE_TESTS_SCENARIO && env.FORCE_TESTS || ''}}
+          fi
```

Comment:
Just to leave some mark and not forget to remove it, should we puts some `echo` message about it?